From 4243c496ab7c5df8d2883ca696f42852d68174e5 Mon Sep 17 00:00:00 2001
From: Eyal Shapira <eyal@wizery.com>
Date: Sun, 18 Dec 2011 09:18:20 +0200
Subject: [PATCH 30/32] Remove ARP wowlan wakeup pattern

As FW now includes Auto ARP response there's no
need to wakeup on ARPs

Signed-off-by: Eyal Shapira <eyal@wizery.com>
---
 src/drivers/driver_nl80211.c |    9 ---------
 1 files changed, 0 insertions(+), 9 deletions(-)

diff --git a/src/drivers/driver_nl80211.c b/src/drivers/driver_nl80211.c
index 5b6df3e..81a66b2 100644
--- a/src/drivers/driver_nl80211.c
+++ b/src/drivers/driver_nl80211.c
@@ -6918,15 +6918,6 @@ static struct rx_filter rx_filters[] = {
 	 .mask_len = 1,
 	},
 
-	{.name = "arp",
-	 .pattern = {0   , 0   , 0   , 0   , 0   , 0   , 0   , 0   ,
-		     0   , 0   , 0   , 0   , 0x08, 0x06},
-	 .pattern_len = 14,
-	 .mask = { 0,                                 /* OCTET 1 */
-		   BIT(4) | BIT(5) },                 /* OCTET 2 */
-	 .mask_len = 2,
-	},
-
 	{.name = "dhcp",
 	 .pattern = {0   , 0   , 0   , 0   , 0   , 0   , 0   , 0   ,
 		     0   , 0   , 0   , 0   , 0   , 0   , 0x45, 0   ,
-- 
1.7.0.4

